draft-ietf-quic-transport.md

```diff
@@ -2635,12 +2638,18 @@ Too Big messages.
 provisional until QUIC's loss detection algorithm determines that the packet is
 actually lost.
 
+
 # Streams: QUIC's Data Structuring Abstraction {#streams}
 
-Streams in QUIC provide a lightweight, ordered, and bidirectional byte-stream
-abstraction modeled closely on HTTP/2 streams {{?RFC7540}}.
+Streams in QUIC provide a lightweight, ordered byte-stream abstraction.
+
+There are two basic types of stream in QUIC.  Unidirectional streams carry data
+in one direction only; bidirectional streams allow for data to be sent in both
+directions.  Different stream identifiers are used to distinguish between
+unidirectional and bidirectional streams, as well as to create a separation
+between streams that are initiated by the client and server (see {{stream-id}}).
 
-Streams can be created either by the client or the server, can concurrently send
+Either type of stream can be created by either endpoint, can concurrently send
 data interleaved with other streams, and can be cancelled.
 
```

Comment:
Not entirely clear which endpoint can cancel.

Reply:
That's not important at this point.